build/Dockerfile

```diff
@@ -99,10 +99,13 @@ LABEL name="NGINX Ingress Controller" \
 	summary="The Ingress controller is an application that runs in a cluster and configures an HTTP load balancer according to Ingress resources." \
 	io.openshift.tags="nginx,ingress-controller,ingress,controller,kubernetes,openshift"
 
-RUN microdnf --nodocs install -y shadow-utils ca-certificates \
+RUN microdnf --nodocs install -y shadow-utils ca-certificates systemd systemd-libs \
 	&& groupadd --system --gid 101 nginx \
 	&& useradd --system --gid nginx --no-create-home --home-dir /nonexistent --comment "nginx user" --shell /bin/false --uid 101 nginx
 
+# temporary fix for CVE-2021-33910 - systemd-pam
+RUN microdnf --nodocs install -y systemd-pam || echo "Can't install systemd-pam - package not available in ubi7"
```

Comment:
should this be moved to line 85, so it runs only for ubi8?

Reply:
Good point 👍🏼